### ISSUE TITLE
cdkDragDisabled not updating while hold click

### ISSUE THREAD

**Issue by EmersonBC** [angular/components#15957]
#### What is the expected behavior?
I'm developing a long press feature in my app that need to allow the user to drag the item only after a period of time holding it.

#### What is the current behavior?
When i start the touch event with the cdkDragDisabled  as true, and after some time holding it change the variable to false, the drag should be working but it is still disabled.

**Comment by jonathanRinciari** [comment]
Any chance you can spin up a stackblitz of an example of your implementation?

**Comment by crisbeto** [comment]
It would be nice to see a Stackblitz, but what I'm guessing is happening is that your event either isn't triggering change detection or is happening after the CDK's event. For what it's worth, you can add a delay to the dragging via the `cdkDragStartDelay` input.

**Comment by EmersonBC** [comment]
I think the cdkDragStartDelay is not implemented in version 7.3.7.

**Comment by mmalerba** [comment]
If this is still a problem, please open a new issue with reproduction steps
